### PR TITLE
Fixes #2870 - Microwaves no longer break forever. Gibber meat no longer collects under the gibber.

### LINKED ISSUE #2870: Meat can appear under the gibber when it spits out meat.
minor oversight, you can still pick up the meat though, just makes no sense as it spews out from the front of the gibber

### PR DESCRIPTION
**Pull request by ghost**
Also cleans up some unrequired src.
Also stops silent failing when trying to shove someone inside of a gibber if someone is inside.

### CODE REVIEW

File: code/modules/food&drinks/kitchen machinery/gibber.dm [hunk 1]
```diff
@@ -266,10 +268,11 @@
 		playsound(src.loc, 'sound/effects/splat.ogg', 50, 1)
 		operating = 0
 		for (var/i=1 to meat_produced)
-			var/list/nearby_turfs = orange(3, get_turf(src))
 			var/obj/item/meatslab = allmeat[i]
+			var/turf/Tx = locate(x - rand(1,3), y, z)
```
Comment (Carbonhell) [inline code/modules/food&drinks/kitchen machinery/gibber.dm:272]: isn't this an in world check
blee

Comment (ghost) [reply code/modules/food&drinks/kitchen machinery/gibber.dm:272]: I did add a check to make sure the turf exists before throwing at it.

Comment (ghost) [reply code/modules/food&drinks/kitchen machinery/gibber.dm:272]: If you have a preferable way to handle this, I'd appreciate it.

**Comment by Spacedong** [comment]
bugfix merge dis

**Comment by Spacedong** [comment]
whats the status on this

**Comment by Kokojos** [comment]
Sure, after a code check +1

**Comment by Spacedong** [comment]
@Kokojos 
do we literally have anybody to codecheck this, carbon already critiqued this, who else is there

**Comment by blobbernaut** [comment]
Code looks fine to me for what it's worth

**Comment by SplergeBucket** [comment]
no one else can codecheck if carbo said yes so merge eet

**Pull request merged**